website/www/site/content/en/contribute/release-guide.md

```diff
@@ -532,6 +530,7 @@ help with this step. Please email `dev@` and ask a member of the `beammaintainer
   1. Create a PR to update beam-site, changes includes:
      * Copy python doc into beam-site
      * Copy java doc into beam-site
+     * **NOTE**: Do not merge this PR until after an RC has been approved.
```

Comment:
```suggestion
     * **NOTE**: Do not merge this PR until after an RC has been approved (see "Finalize the Release").
```

Optional addition to make it clear that people don't need to keep track of this themselves